lib/dennis/zone.rb

```diff
@@ -201,6 +209,17 @@ def verify_nameservers
       true
     end
 
+    def verify
+      req = @client.api.create_request(:post, 'zones/:zone/verify')
+      req.arguments['zone'] = { id: id }
+      @hash = req.perform.hash['zone']
+      verified?
+    rescue ApiaClient::RequestError => e
+      raise unless e.code == 'zone_already_verified'
+
+      true
+    end
```

Comment:
The logic in this method is almost identical as `verify_nameservers` above, it just hits a different endpoint and rescues a different error code.